modules/monitoring/manifests/checks/accounts_slos.pp

```diff
@@ -0,0 +1,31 @@
+# == Class: monitoring::checks::accounts_slos
+#
+# Nagios alerts for Account Team SLOs
+#
+# === Parameters
+#
+# [*enabled*]
+#   This variable enables to define whether to perform Account SLO checks. [This is set to 'true' by default.]
+class monitoring::checks::accounts_slos (
```

Comment:
We'll need to include this class here: https://github.com/alphagov/govuk-puppet/blob/main/modules/monitoring/manifests/checks.pp#L42